README.md

```diff
@@ -420,47 +428,46 @@ specific to the sensor you are using.
+sensor will first attempt to load `configs/onboard_sensor_calibration.json`, and fall
+back to `configs/sensor_calibration.json` if the first option fails. Next, SCOS determines
+whether the loaded calibration data is expired, based on the threshold set by the
+CALIBRATION_EXPIRATION_LIMIT threshold. If calibration data is expired, the sensor will
+attempt to run the calibration action defined by STARTUP_CALIBRATION_ACTION.
+
+SCOS Sensor also supports an additional calibration file, called a differential
+calibration. The differential calibration is used to provide additional scaling factors
+to shift the reference point of data from the onboard calibration terminal to elsewhere
+in the signal path. For instance, a differential calibration file can be provided with
+scaling factors to shift the data reference point from the onboard calibration terminal
+to the antenna port. The differential calibration is loaded separately, and used in
+addition to, either the onboard or lab-provided sensor calibration file.
+
+#### Calibration File Contents
+
+In sensor calibration files, the unit of calibration data is defined by the
+[NTIA-Sensor SigMF Calibration Object](https://github.com/NTIA/sigmf-ns-ntia/blob/master/ntia-sensor.sigmf-ext.md#08-the-calibration-object).
+In differential calibration files, the only key in the calibration data is `loss`.
 
 The `calibration_parameters` key lists the parameters that will be used to obtain
-the calibration data. In the case of the default calibration, there are no
-`calibration_parameters` so the calibration data is found directly within the
-`calibration_data` element and by default scos-sensor will not apply any additional
-gain. Typically, a sensor would be calibrated at particular
-sensing parameters. The calibration data for specific parameters should be listed
-within the calibration_data object and accessed by the values of the settings
-listed in the calibration_parameters element. For example, the calibration below
-provides an example of a sensor calibrated at a sample rate of 14000000.0 samples
-per second at several frequencies with a signal analyzer reference level setting of -25.
+the calibration data. In the case of onboard calibration being generated from scratch,
+the startup calibration action's signal analyzer settings are used as the `calibration_parameters`.
+The calibration data is found directly within the `calibration_data` element and by
+default SCOS Sensor will not apply any additional gain. Typically, a sensor would be
+calibrated at particular sensing parameters. The calibration data for specific parameters
+should be listed within the `calibration_data` object and accessed by the values of the
+settings listed in the calibration_parameters element. For example, the sensor
+calibration below provides an example of a sensor calibrated at a sample rate of
+140000000 samples per second at several frequencies with a signal analyzer reference
+level setting of -25.
```

Comment:
Consider mentioning preamp_enable and attenuation settings as these are also in the example below

Reply:
See my other comment above- addressed this in 2eb839f4eec5d249aa2661b16b5e5a4103382b2f by modifying the example